src/analytics/Properties.tsx

```diff
@@ -638,17 +638,18 @@ interface SendEventsProperties {
     error: string
   }
   [SendEvents.token_dropdown_opened]: {
-    currentTokenAddress: string
+    currentTokenId: string
```

Comment:
@MuckT what do you think about adding the token id column and changing the token address columns to string | null, rather than dropping token address columns? That way if there are dashboards that group by token address, they will continue to have some utility in helping us analyze everything except ETH sends, rather than immediately breaking

Reply:
thinking more about analytics, I think it's very likely that we'll want to group by networkId too

Reply:
so it would be good to add that as well

Reply:
Fixed in https://github.com/valora-inc/wallet/pull/4242/commits/f9d9276183d26c2e7d9795422fbca647bf580498!